backend/src/middleware/handleHtmlContent/handleContentData.js

```diff
@@ -0,0 +1,69 @@
+import extractMentionedUsers from './notifications/extractMentionedUsers'
+import extractHashtags from './hashtags/extractHashtags'
```

Comment:
> [<img alt="ulfgebhardt" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ulfgebhardt) **Authored by [ulfgebhardt](https://github.com/ulfgebhardt)**
_<time datetime="2019-07-01T22:34:47Z" title="Tuesday, July 2nd 2019, 12:34:47 am +02:00">Jul 2, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,69 @@
+import extractMentionedUsers from './notifications/extractMentionedUsers'
+import extractHashtags from './hashtags/extractHashtags'
```


Hashtags are in the folder `hashtags` not sure this is appropriate but its certainly better then in notifications

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-02T06:13:24Z" title="Tuesday, July 2nd 2019, 8:13:24 am +02:00">Jul 2, 2019</time>_

---


I find the locations kind of clear. @ulfgebhardt

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-07-02T06:13:24Z" title="Tuesday, July 2nd 2019, 8:13:24 am +02:00">Jul 2, 2019</time>_

---


I find the locations kind of clear. @ulfgebhardt